src/index.html

```diff
@@ -1,13 +1,427 @@
+                Audio sunglasses
+              </a>
+            </div>
+          </div>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--5-8">
+              <a href="#" class="button">
+                View all
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section testimonials" id="testimonials">
+        <div class="container">
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-5">
+              <h2 class="testimonials__title">
+                Why buy direct
+                <br>
+                from Bose
+              </h2>
+            </div>
+
+            <div class="grid__item grid__item--6-10">
+              <p class="testimonials__paragraph">
+                Free 2-day shipping and returns
+              </p>
+              <p class="testimonials__paragraph">
+                90-day risk-free trial
+              </p>
+              <p class="testimonials__paragraph">
+                World class customer service
+              </p>
+              <p class="testimonials__paragraph">
+                My Bose account management
+              </p>
+              <span class="testimonials__span">
+                A great product is more than what's in the box.
+                It's a promise of premium performance, world-class support, and everything you expect from a trusted brand.
+                It's just one of many reasons why you'll shop with confidence on Bose.com.
+              </span>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">Contact us</h2>
+
+          <div class="grid grid--desktop">
+            <div class="
+              grid__item
+              grid__item--tablet-1-3
+              grid__item--desktop-1-5">
+              <form action="#" class="form">
+                <label>
+                  <input
+                    name="name"
+                    type="text"
+                    class="contact-us__field form-field"
+                    placeholder="Name"
+                    id="name"
+                    required
+                  >
+                </label>
+
+                <label>
+                  <input
+                    name="email"
+                    type="email"
+                    class="contact-us__field form-field"
+                    placeholder="Email"
+                    id="email"
+                    required
+                  >
+                </label>
+
+                <label>
+                  <textarea
+                    name="message"
+                    class="contact-us__field form-field form-field--text"
+                    placeholder="Message"
+                    required
+                  >
+                  </textarea>
```

Comment:
```suggestion
                  <textarea
                    name="message"
                    class="contact-us__field form-field form-field--text"
                    placeholder="Message"
                    required
                  ></textarea>
```

To fix redundant spaces